apps/mull-ui/src/app/components/subnavigation-bar/subnavigation-bar.scss

```diff
@@ -1,8 +1,23 @@
 @import '../../../variables.scss';
 @import '../../../styles.scss';
 
-.subnavigation-container {
-  margin-top: 1rem;
+.subnavigation-header-container {
+  background-color: $background-color;
+  box-shadow: 0 0.25rem 0.25rem rgba(0, 0, 0, 0.1);
+  height: 2.5rem;
+  left: 0;
+  margin-top: -1rem;
+  position: fixed;
+  width: 100%;
+  z-index: 1;
+}
+
+.subnavigation-text-container {
+  position: relative;
+  margin-top: 0.5rem;
+  margin: 0 auto;
+  align-items: center;
+  width: 100%;
   display: flex;
   justify-content: space-evenly;
 }
```

Comment:
This styling is really hacky, what you've done is that you're hiding the box shadow of the top navigation bar with the -1rem top margin. I think it would be easier if we make it part of the top navbar and make it appear only when on the discover page.

This would be a bit complicated however since the top nav bar is the navbar on desktop and the header on mobile. Hence I think we might as well refactor those too so they make more sense. One should always be at the top, and the other should be at the bottom, only appearing on mobile.

What do you think @r-saba

Reply:
I agree @TheGreatMarkus don't think it would be complicated to do either

Reply:
I know we discussed this and I was told that the navigation should look like the one in desktop on both desktop and mobile and that the mockups would be updated to match. Did this change @osahtout ? Not that I mind but I would have done what @TheGreatMarkus mentioned then.

Reply:
I think there's a huge misunderstanding from me and I must apologize. I honestly thought that it was meant that the sub nav would be a separate component as code instead of it being separate visually from the top-nav (unless there was another discussion that I can't remember). 

I think having the sub nav flushed with the scrollable page would cause the user to have difficulty distinguishing the nav and the event card. 

![image](https://user-images.githubusercontent.com/44760251/102914422-3d35b580-444e-11eb-9981-4d9c1306c962.png)

Reply:
The way I did it in my test branch isn't ideal either for the reason you mentioned, and it wasn't the approach I was initially going for. Let's discuss the way to tackle this problem during the daily

Reply:
To document, my approach is now to have the sub nav bar separate from the top navbar. However, the box shadow for the top navbar is disabled and instead place on the sub nav bar, giving the illusion that they are together.

Reply:
To clarify, I've changed back to something similar to what omar did, my initial approach ended up brining too much complexity to the styling and possibly make it easy to break in the future if the page layout needs to change.